plugins/extractors/couchdb/couchdb.go

```diff
@@ -43,6 +43,7 @@ var sampleConfig = `
 type Extractor struct {
 	client      *kivik.Client
 	db          *kivik.DB
+	ctx         context.Context
```

Comment:
can we get by not storing context as internal state?

it is okay to keep passing context

Reply:
okay